eng/pipelines/coreclr/jitstressregs.yml

```diff
@@ -20,24 +20,31 @@ jobs:
   parameters:
     jobTemplate: /eng/pipelines/common/build-coreclr-and-libraries-job.yml
     buildConfig: checked
-    platformGroup: all
+    platforms:
+    - Linux_arm
+    - Linux_arm64
+    - Linux_x64
+    - Windows_NT_x64
+    - Windows_NT_x86
+    # Currently no Windows arm32, Windows arm64 hardware in Helix. https://github.com/dotnet/runtime/issues/1097
+    #- Windows_NT_arm
+    #- Windows_NT_arm64
     jobParameters:
       testGroup: jitstressregs
 
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-test-job.yml
+    jobTemplate: /eng/pipelines/coreclr/templates/test-job.yml
     buildConfig: checked
-    platformGroup: all
-    jobParameters:
-      testGroup: jitstressregs
-      liveLibrariesBuildConfig: Release
-
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/run-test-job.yml
-    buildConfig: checked
-    platformGroup: all
+    platforms:
+    - Linux_arm
+    - Linux_arm64
+    - Linux_x64
```

Comment:
Why not test in OSX_x64 instead as we're covering Unix with the arm platforms?

Reply:
As I understand it, the Linux machines are Azure VMs and not physical hardware, so we don't have to worry about machine resources. There's no interesting difference from the perspective of JIT stress testing to use macOS versus Linux.

Reply:
Ok, then sounds good to me. Thanks for clarifying.